src/shared/lib/validation/opportunity/sprint-with-us.ts

```diff
@@ -0,0 +1,274 @@
+import { uniq } from 'lodash';
+import { dateToMidnight, getNumber, getString, getStringArray, isDateInThePast } from 'shared/lib';
+import SKILLS from 'shared/lib/data/skills';
```

Comment:
This module was originally intended for the CWU skills dropdown. Can you duplicate this module as `capabilities` and import that instead?

Reply:
Sure.  I've also update the list to be consistent with the wireframes/designs.